voorhoede/tudelft-spacefinder
# feat: colored map markers

# Changes

- map markers are now displayed in color depending on building occupancy

# Associated issue

Resolves https://trello.com/c/XvVHSnc5/16-display-occupancy-on-map

# How to test

- Click on a map marker and see that the icon for the selected space corresponds to the marker color
- Wait for / force the data to change and see the marker change its color

# Checklist
